packages/portal/spec-renderer/src/components/SpecDetails.vue

```diff
@@ -25,6 +25,8 @@ import '@kong-ui-public/swagger-ui-web-component'
 import type { SwaggerUIElement } from '@kong-ui-public/swagger-ui-web-component'
 import { PropType, computed, ref, watch, onMounted } from 'vue'
 import { SpecDocument, OperationListItem } from '../types'
+import { createI18n } from '@kong-ui-public/i18n'
+import english from '../locales/en.json'
```

Comment:
suggestion: instead of doing this in every component, you can initialize a single composable as seen here: https://github.com/Kong/shared-ui-components/blob/main/packages/konnect-app-shell/src/composables/useI18n.ts 

and then utilize this in all of your components as shown here: https://github.com/Kong/shared-ui-components/blob/36c16bb4944a74b9e79588edcb664d21d1e733f2/packages/konnect-app-shell/src/components/forms/GeoSelectForm.vue#L74

Reply:
Awesome 👀